_posts/2023-11-02-election-competitiveness-in-charlotte-north-carolina/election-competitiveness-in-charlotte-north-carolina.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "Election Competitiveness in Charlotte, North Carolina"
+description: |
+  Analyzing changes in competitiveness for mayoral and city council elections from 2015 to 2022 in Charlotte, North Carolina.
+author: Angeline Zhao
+date: 2023-11-02
+output:
+  distill::distill_article:
+    self_contained: false
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+## City Council in Charlotte
+
+The Charlotte City Council is made of the Mayor, four at-large city council members, and one council member for each of the seven city council districts. Currently, two members of the city council are Republicans and the remaining are Democrats.
```

Comment:
You might consider visualizing the existing plan with a map! That way, the first thing readers see won't be the boxplot but instead a map, which can be easier to understand.